apps/performance-tests/tsconfig.json

```diff
@@ -4,7 +4,9 @@
     "outDir": "./lib",
     "sourceMap": true,
     "esModuleInterop": true,
-    "allowJs": true
+    "allowJs": true,
+    "declaration": true,
```

Comment:
Is this needed?

Reply:
hm, I'm pretty sure I needed this for debugging *.ts rather than *.js, but I just checked this and it seems the change is not necessary - reverted.